xblock/utils/resources.py

```diff
@@ -22,7 +22,7 @@ def load_unicode(self, resource_path):
         Gets the content of a resource
         """
         package_name = importlib.import_module(self.module_name).__package__
-        return importlib.resources.files(package_name).joinpath(resource_path).read_text()
+        return importlib.resources.files(package_name).joinpath(resource_path.lstrip('/')).read_text()
```

Comment:
Thanks for the `lstrip` fix, nice! It took me some time to understand the fix, though. I had to go into the lti_consumer block and notice that they are using absolute paths, which pkg_resources happily converted to a relative path, but which causes an exception with importlib, so you must strip off the leading slash yourself.

To help out future maintainers, please leave a brief comment explaining why you are stripping the leading slash here.